Tests/KeyboardAutoNavigatorTests.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  KeyboardAutoNavigatorTests.swift
+//  KeyboardSupport-iOSTests
+//
+//  Created by John Davis on 12/18/18.
+//  Copyright © 2018 Bottle Rocket. All rights reserved.
+//
+
+import XCTest
+@testable import KeyboardSupport
+
+class KeyboardAutoNavigatorTests: XCTestCase {
+
+    // MARK: - Properties
+    
+    private var keyboardToolbar: KeyboardToolbar!
+    private var keyboardNavigator: KeyboardAutoNavigator?
+    private var delegateMock: MockKeyboardAutoNavigatorDelegate?
+    
+    // MARK: - Tests
+    
+    override func setUp() {
+        super.setUp()
+        delegateMock = MockKeyboardAutoNavigatorDelegate()
+        keyboardToolbar = KeyboardToolbar()
+        keyboardNavigator = KeyboardAutoNavigator(containerView: UIView(), defaultToolbar: keyboardToolbar, returnKeyNavigationEnabled: true)
+        keyboardNavigator?.delegate = delegateMock
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        delegateMock = nil
+        keyboardToolbar = nil
+        keyboardNavigator = nil
+    }
+
+    func test_keyboardAutoNavigator_invokesDelegateOnNext() {
+        keyboardNavigator?.keyboardAccessoryDidTapNext(keyboardToolbar)
+        XCTAssertEqual(delegateMock?.keyboardNavigatorDidTapNextCount, 1)
+        XCTAssertTrue(delegateMock?.keyboardNavigatorDidTapNextLastNavigator === keyboardNavigator )
```

Comment:
Extra space before the closing parenthesis on these 3 `XCTAssertTrue()`s.

Reply:
https://github.com/BottleRocketStudios/iOS-KeyboardSupport/pull/36/commits/0f6198470a1e1dccdb7a2eab933fafabc78d42a4